### PR TITLE
Spanish traduction added

### PR DESCRIPTION
**Pull request by SalvadorAngel**
General **spanish traduction** was added valid for most regions.

### CODE REVIEW

File: ananas/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <string name="iamutkarshtiwari_github_io_ananas_saturation">Saturacion</string>
+    <string name="iamutkarshtiwari_github_io_ananas_save_error">Error en guardado!</string>
+    <string name="iamutkarshtiwari_github_io_ananas_saving_image">Guardando...</string>
+    <string name="iamutkarshtiwari_github_io_ananas_size">Tamaño</string>
+    <string name="iamutkarshtiwari_github_io_ananas_smooth">Suavidad</string>
+    <string name="iamutkarshtiwari_github_io_ananas_stickers">Stickers</string>
+    <string name="iamutkarshtiwari_github_io_ananas_text">Texto</string>
+    <string name="iamutkarshtiwari_github_io_ananas_white_skin">Tono de piel</string>
+    <string name="materialcolorpicker__btnSelectColor">Seleccionar</string>
+    <string name="materialcolorpicker__errHex">Formato de Codigo Hex incorrecto</string>
```
Comment (iamutkarshtiwari) [inline ananas/src/main/res/values-es/strings.xml]: Please add a new line feed.

Comment (iamutkarshtiwari) [inline ananas/src/main/res/values-es/strings.xml]: Please add tests for these as well: 
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L98-L108
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L41-L54
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L80-L87

Comment (SalvadorAngel) [reply ananas/src/main/res/values-es/strings.xml]: Done.

Comment (SalvadorAngel) [reply ananas/src/main/res/values-es/strings.xml]: Done.

**Pull request merged**